code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -189,20 +189,20 @@
 	else if(istype(target, /obj/machinery/door/airlock))
 		var/obj/machinery/door/airlock/A = target
 
-		if(!A.requiresID() || A.allowed(user)) //This is to prevent stupid shit like hitting a door with an arm blade, the door opening because you have acces and still getting a "the airlocks motors resist our efforts to force it" message.
+		if((!A.requiresID() || A.allowed(user)) && A.hasPower()) //This is to prevent stupid shit like hitting a door with an arm blade, the door opening because you have acces and still getting a "the airlocks motors resist our efforts to force it" message, power requirement is so this doesn't stop unpowered doors from being pried open if you have access
 			return
 		if(A.locked)
 			to_chat(user, "<span class='warning'>The airlock's bolts prevent it from being forced!</span>")
 			return
 
 		if(A.hasPower())
```

Comment:
This is the same check that you just put in 192 (that instantly returns) so when would this ever run?

Reply:
Previously, if you attempted to pry open a door that your ID had access to/had no access requirement, this would trigger a return and cancel the opening. However, if the door was unpowered, and thus unopenable, it would still prevent prying it open.

Reply:
Actually, theres probably a better way to do this, I'll modify it

Reply:
don't respond to coffee-lacking posts and just put the laughing emoji instead.

This still works, it can just be written better.